go/pkg/client/tree.go

```diff
@@ -536,17 +539,17 @@ func (c *Client) ComputeOutputsToUpload(execRoot, workingDir string, paths []str
 			continue
 		}
 		// A directory.
-		fs := make(map[string]*fileSysNode)
-		if e := loadFiles(absPath, "", "", nil, []string{"."}, fs, cache, treeSymlinkOpts(c.TreeSymlinkOpts, sb)); e != nil {
+		fsn := make(map[string]*fileSysNode)
```

Comment:
nit: I think this can be fs now since you don't use the filesystem package anymore?

Reply:
done.